### PR TITLE
update render fidelity main page to list renderers in a table.

### PR DESCRIPTION
**Pull request by bhouston**
Updated the homepage of the render fidelity page to have a list of renders in a table along with their main capabilities, and how they are tested:

Screenshot 2023-10-25 at 9 02 58 AM

### CODE REVIEW

File: packages/render-fidelity-tools/test/results-viewer.html [hunk 1]
```diff
@@ -103,24 +103,72 @@ <h2>Render Fidelity Comparison Results</h2>
...
+          target="_blank" rel="noopener">gltf-sample-viewer</a></td>
+          <td>Real-Time</td>
+          <td>Web</td>
+    </tr> 
+    <tr>
+      <td><a
+        href="https://www.3ds.com/products-services/3dexcite/resource-center/stellar-physically-correct/"
+        target="_blank" rel="noopener">Dassault STELLAR</a></td>
+    <td>Path Tracing</td>
+    <td>Unknown</td>
```
Comment (elalish) [inline packages/render-fidelity-tools/test/results-viewer.html:145]: @bsdorra Care to tell us about the platforms STELLAR runs on?

**Review by elalish — DISMISSED** [review]
Great idea, thanks!

File: packages/render-fidelity-tools/test/results-viewer.html (hunk 1 above)
Comment (bsdorra) [reply packages/render-fidelity-tools/test/results-viewer.html:145]: Linux and Windows 

More specifically:
GPU (x86_64 host only): Maxwell, Pascal, Volta, Turing, Ampere, Ada
CPU: 
AVX2, AVX512 
SSE2, SSE41, AVX (legacy render modes only)
ARMv8.2-A

Comment (bhouston) [reply packages/render-fidelity-tools/test/results-viewer.html]: I've updated the page to include the Stellar platform info!

**Comment by bhouston** [comment]
Thank you! Sweet!

**Pull request merged**